packages/core/admin/admin/src/components/GuidedTour/layout.js

```diff
@@ -0,0 +1,153 @@
+        },
+        type: 'REDIRECT',
+        target: '/content-manager',
+      },
+    },
+    create: {
+      title: {
+        id: 'app.components.GuidedTour.CM.create.title',
+        defaultMessage: '⚡️ Create sample content',
+      },
+      content: {
+        id: 'app.components.GuidedTour.CM.create.content',
+        defaultMessage:
+          '<p>Create and manage all the content here in the Content Manager.</p><p>Ex: Taking the blog website example further, one can write an article, save and publish it as they like.</p>',
+      },
+      cta: {
+        title: {
+          id: 'app.components.GuidedTour.CM.create.ctaTitle',
+          defaultMessage: 'Create sample content',
+        },
+        type: 'CLOSE',
+      },
+    },
+    success: {
+      title: {
+        id: 'app.components.GuidedTour.CM.success.title',
+        defaultMessage: 'Step 2: Completed ✅',
+      },
+      content: {
+        id: 'app.components.GuidedTour.CM.success.content',
+        defaultMessage: '<p>Awesome, one last step to go!</p><b>🚀  See content in action</b>',
+      },
+      cta: {
+        title: {
+          id: 'app.components.GuidedTour.CM.success.cta.title',
+          defaultMessage: 'Test the API',
+        },
+        type: 'REDIRECT',
+        target: '/settings/api-tokens',
+      },
+    },
+  },
+  apiTokens: {
+    home: {
+      title: {
+        id: 'app.components.GuidedTour.apiTokens.create.title',
+        defaultMessage: '🚀 See content in action',
+      },
+      cta: {
+        title: {
+          id: 'app.components.GuidedTour.home.apiTokens.cta.title',
+          defaultMessage: 'Test the API',
+        },
+        type: 'REDIRECT',
+        target: '/settings/api-tokens',
+      },
+    },
+    create: {
+      title: {
+        id: 'app.components.GuidedTour.apiTokens.create.title',
+        defaultMessage: '🚀 See content in action',
+      },
+      content: {
+        id: 'app.components.GuidedTour.apiTokens.create.content',
+        defaultMessage:
+          '<p>Generate an authentication token here and retrieve the content you just created.</p>',
+      },
+      cta: {
+        title: {
+          id: 'app.components.GuidedTour.apiTokens.create.cta.title',
+          defaultMessage: 'Generate an API Token',
+        },
+        type: 'CLOSE',
+      },
+    },
+    success: {
+      title: {
+        id: 'app.components.GuidedTour.apiTokens.success.title',
+        defaultMessage: 'Step 3: Completed ✅',
+      },
+      content: {
+        id: 'app.components.GuidedTour.apiTokens.success.content',
+        defaultMessage:
+          "<p>See content in action by making an HTTP request:</p><ul><li><p>To this URL: <light>https://'<'YOUR_DOMAIN'>'/api/'<'YOUR_CT'>'</light></p></li><li><p>With the header: <light>Authorization: bearer '<'YOUR_API_TOKEN'>'</light></p></li></ul><p>For more ways to interact with content, see the <documentationLink>documentation</documentationLink>.</p>",
```

Comment:
```suggestion
          "<p>See content in action by making a HTTP request:</p><ul><li><p>To this URL: <light>https://'<'YOUR_DOMAIN'>'/api/'<'YOUR_CT'>'</light></p></li><li><p>With the header: <light>Authorization: bearer '<'YOUR_API_TOKEN'>'</light></p></li></ul><p>For more ways to interact with content, see the <documentationLink>documentation</documentationLink>.</p>",
```

Also: could the link to the API be a real link, to the user can click on it?

Reply:
Grammarly seems to recommend using "an HTTP" rather than "a HTTP" :D

(For the link, I'm not sure what can be done, maybe @vincentbpro knows)

Reply:
I would go for "an", not sure though [](https://www.supaproofread.com/an-hotel-or-a-hotel.html)

Reply:
[https://www.supaproofread.com/an-hotel-or-a-hotel.html](https://www.supaproofread.com/an-hotel-or-a-hotel.html)

Reply:
Fine for me. So the one remaining question would be about the link. @vincentbpro opinions?

Reply:
Sorry missed that :)

Reply:
Ah, alright. I thought that these are placeholders and would be replaced. I'm wondering whether we should then rather link to `/admin/settings/users-permissions/roles/2` because there you can already see the URL of the endpoint + you can add permissions to view the route. Might be easier for the beginning than adding HTTP headers, but I guess that's too big of a discussion for now, so feel free to close this thread.

Reply:
That would be easier for sure :), but 1/3 steps in the guided tour are related to creating an API token. Guiding users to a way to request the API without using this token may be confusing / would require a complete rework of the GT. This is a product level question @iparidhi